public/app/app.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './jquery-import';
 import { Provider } from 'react-redux';
-import store, { persistor } from './redux/store';
+import store from './redux/store';
```

Comment:
This is good, but doesn't prevent catching the same issue again. Would you mind adding a linter rule to check for unused imports?

Reply:
Do you mean to use ESlint?  I am glad to add it. I will open a new pull request to introduce the ESlint as the linter.

Reply:
Added an issue here https://github.com/grafana/phlare/issues/672